deploy/pimod/companion.Pifile

```diff
@@ -0,0 +1,14 @@
+# We base companion over the first version, that is generated from the install script
+FROM https://s3.amazonaws.com/downloads.bluerobotics.com/Pi/experimental/companion-docker-15-oct-2021-1.0.0-beta1.img.zip
+
+PUMP 100M
+
+# Start docker and wait a couple of seconds to make sure that is running and ready
+RUN sudo dockerd &
+RUN sleep 10
+
+# Replace current companion version and update bootstrap to start it
+RUN sudo docker images
+RUN sudo docker rmi --force bluerobotics/companion-core:1.0.0.beta1
+RUN sudo docker pull bluerobotics/companion-core:1.0.0.beta4
+RUN sed -i 's/1.0.0.beta1/1.0.0.beta4/g' /root/.config/companion/bootstrap/startup.json
```

Comment:
Just to make sure, this will need to be updated manually for each new release, right?

Reply:
Yes, there is an open issue for that on pimod:
https://github.com/Nature40/pimod/issues/45